### ISSUE TITLE
Identical Crafting Recipes

### ISSUE THREAD

**Issue by xenonca** [za267/Aurelium#1]
The Key and the Monorail Switch have the same crafting recipe which only lets you make Monorail Switches but no keys.

**Comment by za267** [comment]
I started working on this and found the recipes responsible.

https://github.com/za267/Aurelium/blob/202c6a1382b2e2c037b4dd92a10012a76c65d6a5/locks/init.lua#L584

and

https://github.com/za267/Aurelium/blob/202c6a1382b2e2c037b4dd92a10012a76c65d6a5/monorail/nodes_crafts.lua#L45

I've tested a modified recipe for the monorail to try and specify it on the two side rows but it doesn't seem to have worked.

**Comment by za267** [comment]
Here's the link to the commit:

https://github.com/za267/Aurelium/commit/13e63f2aeca657347b8dfd4c8a9961d2cbf2f054

**Comment by xenonca** [comment]
What about just changing the recipe completely, eg. for the monorail, maybe like this:
`    recipe = {
        {"default:stick"},
        {"default:stick"},
        {"default:steel_ingot"},
}`

Of course only if that doesn't affect any other recipe

**Comment by za267** [comment]
Yes I could probably change it completely, at least that would be  a good temporary alternative.  I'm a bit miffed that I can't seem to simply change the alignment... That would allow for much more flexibility.  I should ask that question on Minetest's IRC channel.

**Comment by za267** [comment]
I tested the change and it does work.  I'll make the change permanently on the server.
